examples/private-vpc-no-zone/main.tf

```diff
@@ -0,0 +1,299 @@
+  availability_zone = "us-west-2a"
+}
+
+resource "aws_subnet" "client" {
+  vpc_id                  = aws_vpc.test.id
+  cidr_block              = "10.0.2.0/24"
+  map_public_ip_on_launch = true
+
+  tags              = var.tags
+  availability_zone = "us-west-2a"
+}
+
+resource "aws_internet_gateway" "test" {
+  vpc_id = aws_vpc.test.id
+
+  tags = var.tags
+}
+
+resource "aws_route_table" "test" {
+  vpc_id = aws_vpc.test.id
+
+  route {
+    cidr_block = "0.0.0.0/0"
+    gateway_id = aws_internet_gateway.test.id
+  }
+
+  tags = var.tags
+}
+
+resource "aws_route_table_association" "client" {
+  subnet_id      = aws_subnet.client.id
+  route_table_id = aws_route_table.test.id
+}
+
+resource "aws_route_table_association" "server" {
+  subnet_id      = aws_subnet.server.id
+  route_table_id = aws_route_table.test.id
+}
+
+resource "aws_security_group" "client_sec_group" {
+  name        = "${var.deployment_prefix}-client-sg"
+  description = "client security group"
+  tags        = var.tags
+  vpc_id      = aws_vpc.test.id
+  ingress {
+    description = "Allow outbound to ssh"
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+module "redpanda-cluster" {
+  source                          = "../../"
+  public_key_path                 = var.public_key_path
+  broker_count                    = var.nodes
+  deployment_prefix               = var.deployment_prefix
+  enable_monitoring               = var.enable_monitoring
+  tiered_storage_enabled          = var.tiered_storage_enabled
+  allow_force_destroy             = var.allow_force_destroy
+  vpc_id                          = aws_vpc.test.id
+  distro                          = var.distro
+  hosts_file                      = var.hosts_file
+  tags                            = var.tags
+  associate_public_ip_addr_client = true
+  security_groups_client          = [aws_security_group.client_sec_group.id]
+  subnets                         = {
+    broker = {
+      "us-west-2a" = aws_subnet.server.id
+    }
+    client = {
+      "us-west-2a" = aws_subnet.client.id
+    }
+  }
+  client_count      = 1
+  availability_zone = ["us-west-2a"]
+  egress_rules      = {
+    "SSH" = {
+      description     = "Allow outbound to ssh"
+      from_port       = 22
+      to_port         = 22
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = ["0.0.0.0/0"]
+      self            = null
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "PING" = {
+      description     = "Allow ICMP"
+      from_port       = -1
+      to_port         = -1
+      protocol        = "icmp"
+      enabled         = true
+      cidr_blocks     = ["0.0.0.0/0"]
+      self            = null
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+  }
+  ingress_rules = {
```

Comment:
Didn't notice it before, but I don't see the pandaproxy port listed

Reply:
(more specifically, we should probably have it added for the other examples as well as a common rule)

Reply:
Which port is that?

Reply:
8082